frontend/packages/shipwright-plugin/src/types.ts

```diff
@@ -1,16 +1,26 @@
-import { IBuild } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/Build';
-import { IBuildRun } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/BuildRun';
+import { IBuild as IBuildV1Alpha1 } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/Build';
+import { IBuildRun as IBuildRunV1Alpha1 } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/BuildRun';
+import { IBuild as IBuildv1Beta1 } from '@kubernetes-models/shipwright/shipwright.io/v1beta1/Build';
+import { IBuildRun as IBuildRunv1Beta1 } from '@kubernetes-models/shipwright/shipwright.io/v1beta1/BuildRun';
 import { K8sResourceCondition } from '@console/internal/module/k8s';
 
 // Add missing latestBuild to Build
-export type Build = IBuild & { latestBuild?: BuildRun };
+export type Build =
+  | (IBuildV1Alpha1 & { latestBuild?: BuildRun })
+  | (IBuildv1Beta1 & { latestBuild?: BuildRun });
 
-export type BuildSpec = IBuild['spec'];
+export type BuildSpec = IBuildV1Alpha1['spec'];
 
-export type BuildStatus = IBuild['status'];
+export type BuildStatus = IBuildV1Alpha1['status'];
```

Comment:
Same here:

```suggestion
export type BuildStatus = IBuildV1Alpha1['status'] | IBuildV1Beta1['status'];
```

Reply:
`export type BuildStatus = IBuildV1Alpha1['status'] & IBuildV1Beta1['status'];` as the suggested one gave many typeErrors.